templates/raspbian/resources/usr/share/initramfs-tools/scripts/cattlepi-base/helpers

```diff
@@ -216,13 +216,17 @@ cattlepi_run_bootcode()
 
 cattlepi_fetch_update_images()
 {
+    # Define an array to store md5s of current images
+    local current_imgs=()
+
     cattlepi_fetch_update_config
     for img in initfs rootfs; do
         local md5sum=$(jq -r ".${img}.md5sum" "${CATTLEPI_DIR}/config")
         if [ -r "${CATTLEPI_IMAGES_DIR}/${md5sum}" ]; then
             if [ "${CATTLEPI_MD5SKIP}" != "yes" ]; then
                 echo "${md5sum}  ${CATTLEPI_IMAGES_DIR}/${md5sum}" > /tmp/check_md5
-                md5sum --quiet -c /tmp/check_md5 || rm -rf ${CATTLEPI_IMAGES_DIR}/${md5sum}
+                # Don't add the image's md5sum to the array to be saved if md5sum check fails
+                md5sum --quiet -c /tmp/check_md5 || rm -rf ${CATTLEPI_IMAGES_DIR}/${md5sum} && continue
```

Comment:
i would remove the continue. 
let it flow through and if it cannot download it below it will panic (crashes and reboots).

Reply:
Ah - I see that now.  Will do.